src/components/Label/README.md

```diff
@@ -254,6 +254,28 @@ LANDING_BLOCK-->
 
 <!--/GITHUB_BLOCK-->
 
+## Animation
+
+<!--LANDING_BLOCK
+
+<ExampleBlock
+    code={`
+<Label animation="gradient">Content</Label>
+`}
+>
+    <UIKit.Label animation="gradient">Content</UIKit.Label>
+</ExampleBlock>
+
+LANDING_BLOCK-->
```

Comment:
Is it going to break https://gravity-ui.com/components/uikit/label? :]

Reply:
nvm, I guess landing is using readme from current version of uikit, not from main branch